docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -21,6 +21,9 @@ New Features
 * Worker failure detection and high availability failover occurs within 30 seconds. Pulp processes are
   considered missing after 25 seconds, and heartbeats occur every 5 seconds.
 
+* When removing orphan content, the number of items being that have been removed is now stored as
+  the result on the orphan removal task. This count is broken down by content unit type.
```

Comment:
It depends on what you mean by "reported by pulp." The result is stored on the task which you can look up via `pulp-admin tasks details ...`. The orphan remove command doesn't report the results though (although thinking over it, maybe it should?)